events/schema.py

```diff
@@ -238,6 +240,14 @@ def resolve_ticket_system(self, info, **kwargs):
         # Event object is needed for resolving EventTicketSystem fields
         return self
 
+    def resolve_can_child_enroll(self: Event, info, **kwargs) -> Optional[bool]:
+        child_id = get_node_id_from_global_id(kwargs["child_id"], "ChildNode")
+        try:
+            child = Child.objects.user_can_view(info.context.user).get(id=child_id)
+        except Child.DoesNotExist:
+            return None
```

Comment:
Shouldn't this raise an exception? At least the None does not add anything related to False, so I wonder why the None should ever be returned. A minor thing, but I would like to hear other opinions.

Reply:
Raising an error sounds perfectly good. :+1: